proposals/0264-stdlib-preview-package.md

```diff
@@ -139,33 +137,26 @@ As such, the preview package will remain at major version `0` **permanently**. M
 
 Because we do not have cross-module optimization, package implementations should make use of `@inlinable` annotations. However, these annotations should be in the context of source stability only and should be fully reevaluated by the library integrator when stabilizing the ABI.
 
 ## Alternatives considered
 
-### Single versus Multiple Packages
-
-An alternative to a single monolithic preview package would be multiple packages. For example, each accepted evolution proposal could have its own package.
-
-This would trade convenience for flexibility. In particular, multiple packages would avoid the source-breaking nature of proposal revisions (and the lesser problem of any additions being technically source breaking). But it would mean that adopters of the different previews would need to look up and add individual proposals rather than get them all in one go, and would rarely "discover" new features also in the package they're already importing.
-
-### Simultaneous Preview and Library Addition
+### Additional Review Before Library Integration
 
-If the goal was to facilitate rapid release only, but not a longer feedback period, then the additional review step would not be necessary. Instead, new features would be integrated directly into the package and library simultaneously.
+A previous version of this proposal prescribed a window of time before promoting accepted features into the standard library, to provide additional bake time and an additional review pointfor the feature. However, this approach delays some important feedback that can only be gained from integration into the standard library, where overload resolution and performance can have different outcomes than the same APIs shipped in a package.
 
-However, experience has shown that proposals are not always perfect. Recently, significant if manageable issues were discovered with the API of both [SE-180](https://github.com/apple/swift-evolution/blob/master/proposals/0180-string-index-overhaul.md) and [SE-202](https://github.com/apple/swift-evolution/blob/master/proposals/0202-random-unification.md), and in a near-miss [SE-220](https://github.com/apple/swift-evolution/blob/master/proposals/0220-count-where.md) had to be reverted late during 5.0 convergence due to typechecker performance impact. These issues would likely have been caught by a more prolonged exposure in a package.
+### No Semantic Versioning
 
-### Is a Second Review Necessary?
+An alternative to keeping the semver major version at `0` is to not version the package at all. With this approach, the package could only be included by packages by specifying a branch-based dependendcy. However, this would be too restricting at this time, as a branch-based dependency can only be imported by other branch-based dependencies. This would effectively limit use of the `SwiftPreview` package to top-level applications only.
```

Comment:
Typo: `dependendcy` -> `dependency`

```suggestion
An alternative to keeping the semver major version at `0` is to not version the package at all. With this approach, the package could only be included by packages by specifying a branch-based dependency. However, this would be too restricting at this time, as a branch-based dependency can only be imported by other branch-based dependencies. This would effectively limit use of the `SwiftPreview` package to top-level applications only.
```